src/components/forms/summit-form.js

```diff
@@ -707,6 +707,16 @@ class SummitForm extends React.Component {
                                 </label>
                             </div>
                         </div>
+                        <div className="col-md-4 checkboxes-div">
```

Comment:
this is not correct
BS only supports 12 cols
move [the](https://github.com/fntechgit/summit-admin/blob/ec3ff07a83eedbc8ed57dcf73e458f4a13f4d462/src/components/forms/summit-form.js#L700) to a new row below